lib/event_listeners.js

```diff
@@ -578,9 +578,9 @@ AWS.EventListeners = {
         }
       }
 
+      var time = resp.request.service.getSkewCorrectedDate().getTime();
```

Comment:
This function is the listener for when a request finishes.
i'm not sure what you mean by get the time in a global var.
this gets the time for the specific request that was just made and gets the delta from this request's start time in seconds.
it is used in the message we print for errors,
i.e. [AWS dynamodb 400 0.004s 0 retries]
i just moved the variables outside the message function so it can be used in the message and to see if i need to set the error.